### DISCUSSION TITLE
[Alpha] Introducing Lens, smart search — powered by AI

### DISCUSSION THREAD

**Discussion by SamyPesse** [GitbookIO/community#98] [Announcements]
Hello 👋

We’ve been hard at work finding new ways to rethink and improve the search experience in published content — as you can see from [the recent redesign of the search box](https://changelog.gitbook.com/february-2023/07-february-23-monorepos-customized-commit-messages-and-more). But that improvement was just the start.

One of these explorations leverages recent developments in AI technology — and the semantic content on your GitBook pages — to help readers find answers faster when using your documentation.

Yesterday, we released an alpha version of “Smart Search”. When it’s enabled, readers have the option to ask questions about your published content and get near-instant answers — right in the search box.

[ aisearch.mp4 
](https://user-images.githubusercontent.com/845425/217935164-96b7b545-bea2-4b41-973f-9ec9b259f934.mp4)

# How does it work?
Eventually, organizations on our Pro plan will be able to join the alpha and enable Smart Search from their space customization settings. Once that’s done, there is no step two 🎉 GitBook will automatically index and process your published content, and your readers will see the new **Ask a question** button in the search box.

# Want to join the alpha?
Over time, we’ll gradually onboard organizations on our Pro plan to give them access to this feature. Want to join the alpha? You can [contact our support team](mailto:support@gitbook.com) — they’ll be happy to help.

Requirements: Your organization must be on the Pro plan or be a non-profit/open-source organization.

# Got some feedback?
We want to make this feature the best it can be. If you have feedback that will help us improve it, please add it directly in this thread, or [contact our support team](mailto:support@gitbook.com).

**Comment by petros** [comment]
This looks exciting! Would be great if there was a way to see the pages the AI used to come up with the answer.

**Reply by SamyPesse** [reply]
Planned and will be shipped soon 😉  We want to make sure the readers can understand the sources used to come up with the answer, especially when there is an error margin with it.

**Reply by DooleyCat** [reply]
Thanks so much, @SamyPesse! We had the same request on our end, to please expose the links either as a source list or ideally expose the most helpful/relevant link directly in the body of reply.
